packages/itwinui-react/src/core/TransferList/TransferList.tsx

```diff
@@ -156,7 +162,9 @@ const TransferListListItem = React.forwardRef((props, ref) => {
       onKeyDown={onKeyDown}
       active={active}
       actionable={actionable}
-      tabIndex={disabled ? undefined : -1}
+      tabIndex={-1}
+      aria-disabled={disabled}
```

Comment:
Similar to `role=listbox` on the list, each of the listitems should have `role=option`. Also should have `aria-selected` when it's active.

See APG: https://www.w3.org/WAI/ARIA/apg/patterns/listbox/examples/listbox-rearrangeable/

After adding those, please also test in NVDA to confirm it's working.

Reply:
This has been added, but unit tests were not passing, so I still need to add those

Reply:
The unit test has been added

Reply:
![TransferListNVDA](https://github.com/iTwin/iTwinUI/assets/78564221/6ff553bb-39c1-4b8f-b343-4878ca0f095b)

Here is the NVDA test. Is it supposed to tell me when an item is selected?

Reply:
yes. I think you're missing `aria-multiselectable` (check the APG link above).

here's voiceover on safari with `aria-multiselectable`:

https://github.com/iTwin/iTwinUI/assets/9084735/d138346f-3594-4a4a-8be4-49612de67dcc

Reply:
`aria-multiselectable` has been added. I still need to apply `aria-activedescendant` to the listbox though

![TransferListNVDAWithMultiSelect](https://github.com/iTwin/iTwinUI/assets/78564221/f7d0ec8e-12cc-4b6a-8678-c9ab778fbe6c)

Reply:
`aria-activedescendant` is useful when the focus is on a different element, e.g. in the case of ComboBox, the focus is on input rather than the options. In this case, the focus is actualy on the options, so it should be ok.

You can compare by testing that demo from APG using NVDA as well. The results should be very similar.

Reply:
![TransferListNVDAW3Example](https://github.com/iTwin/iTwinUI/assets/78564221/1418898e-ac8f-4230-a7c7-2a140b5e5930)

The results are very similar